CODE_OF_CONDUCT.md

```diff
@@ -106,7 +106,7 @@ Violating these terms may lead to a permanent ban.
 ### 4. Permanent Ban
 
 **Community Impact**: Demonstrating a pattern of violation of community
-standards, including sustained inappropriate behavior, harassment of an
+standards, including sustained inappropriate behavior,  harassment of an
```

Comment:
```suggestion
standards, including sustained inappropriate behavior, harassment of an
```

Reply:
I just copied it from the Contributor Covenant code of conduct page, not sure if we should modify it. 🤷‍♂️

Reply:
I think a double space could be modified